doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,46 +1,119 @@
-////
-This is the base template for Hazelcast integration module guides.
+:github-address: https://github.com/hazelcast-guides/kubernetes
+:templates-url: templates:ROOT:page$/
+:hazelcast: Hazelcast IMDG
+:framework: Kubernetes
 
-You can clone this repository, work on it and create your guide, and then push to a new repository.
-////
+= Hazelcast for Kubernetes
 
-:github-address: [github-repo-address-of-your-guide]
+This sample is a guideline on how to start Hazelcast cluster on the Kubernetes environment.
 
-:templates-url: https://raw.githubusercontent.com/hazelcast-guides/adoc-templates/master
+include::{templates-url}/link-to-repo.adoc[]
 
-// Use this relative url if you are going to publish the guide on the guides site.
-// Note that this url will not work locally and raise asciidoctor errors.
-// So, complete the guide with the above url and set the below one just before 
-// publishing on the guides site.
-//
-// :templates-url: templates:ROOT:page$/
+== What You’ll Learn
 
-= Title
+In this guide you will deploy Hazelcast cluster to Kubernetes.
 
-// Content entered directly below the header but before the first section heading is called the preamble.
+== Prerequisites
 
-include::{templates-url}/link-to-repo.adoc[]
+1. Up and running https://kubernetes.io/[Kubernetes] version 1.9 or higher.
+* For development and testing, you may use https://github.com/kubernetes/minikube[Minikube]
+* You must have the Kubernetes command line tool, https://kubernetes.io/docs/tasks/tools/install-kubectl/[kubectl], installed
+2. Another important note would be that this document assumes some familiarity with `kubectl`, Kubernetes, and Docker.
 
-== What You’ll Learn
+== Deployment Steps
 
-// Define the problem and tell reader what he/she will learn from this guide.
+=== Creating Role Binding
+Hazelcast uses Kubernetes API to discover nodes and that is why you need to grant certain permissions. The simplest Role Binding file can look as `rbac.yaml`. Note that you can make it more specific, since Hazelcast actually uses only certain API endpoints. Note also that if you use "DNS Lookup Discovery" instead of "REST API Discovery", then you can skip the Role Binding step at all. Read more at https://github.com/hazelcast/hazelcast-kubernetes[Hazelcast Kubernetes API Plugin].
```

Comment:
```suggestion
Hazelcast uses Kubernetes API to discover nodes and that is why you need to grant certain permissions. The simplest Role Binding file looks like `rbac.yaml`. Note that you can make it more specific, since Hazelcast actually uses only certain API endpoints. Note also that if you use "DNS Lookup Discovery" instead of "REST API Discovery", then you can skip the Role Binding step at all. Read more at https://github.com/hazelcast/hazelcast-kubernetes[Hazelcast Kubernetes API Plugin].
```